server/game/cards/03_TWI/units/501stLiberator.ts

```diff
@@ -0,0 +1,25 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { Trait } from '../../../core/Constants';
+
+export default class _501stLiberator extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '7252148824',
+            internalName: '501st-liberator',
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.addWhenPlayedAbility({
+            title: 'If you control another Republic unit, you may heal 3 damage from a base.',
```

Comment:
Since this ability says "may" we need optional: true and need to have a check for a Pass button in the test